pkg/ebtables/ebtables.go

```diff
@@ -65,13 +65,14 @@ func DeleteRuleByDevice(bridge string) error {
 	cmd := exec.Command(cmdebtables, "--list", ChainForward)
 	stdout, err := cmd.CombinedOutput()
 	if err != nil {
-		return fmt.Errorf("Failed to get list of ebtables rules: %v", err)
+		return fmt.Errorf("failed to get list of ebtables rules: %v", err)
 	}
 
 	for _, line := range strings.Split(string(stdout), "\n") {
 		if strings.Contains(line, bridge) {
 			if err = DeleteRule(strings.TrimSpace(line)); err != nil {
-				return fmt.Errorf("Failed to delete rule: %v", err)
+				return fmt.Errorf(
```

Comment:
you don't need to use `%s`, %q could help on that. It encloses the provided param with single quote